openforcefield/utils/toolkits.py

```diff
@@ -1220,7 +1220,10 @@ def compute_partial_charges(self, molecule, quantum_chemical_method="AM1-BCC", p
 
     def compute_partial_charges_am1bcc(self, molecule):
         """
-        Compute AM1BCC partial charges with OpenEye quacpac
+        Compute AM1BCC partial charges with OpenEye quacpac. This function will attempt to use
+        the OEAM1BCCELF10 charge generation method, but may print a warning and fall back to
+        normal OEAM1BCC if an error is encountered.
```

Comment:
Do you want to give a "such as" example for when this might occur? 

Also, "may print a warning and fall back..." is a bit ambiguous. Does it only do this sometimes? Always? What triggers doing this/not doing it? If it's simple might be better to explain.

Reply:
Agreed! A SMILES example or two would be useful.